weave/trace_server/file_storage_credentials.py

```diff
@@ -0,0 +1,115 @@
+import base64
+from typing import Optional, TypedDict, Union
+
+from azure.core.credentials import TokenCredential
+from google.oauth2.credentials import Credentials as GCPCredentials
```

Comment:
This is not safe.  You need to try except import here because users may not have any of these.

Reply:
this is server code, not user code